Jenkinsfile-insertions

```diff
@@ -0,0 +1,88 @@
+import groovy.json.JsonSlurper
+
+pipeline{
+	agent any
+
+	stages{
+		stage('Check upstream builds succeeded'){
```

Comment:
Maybe this could read "Check AddLinks Download and Orthoinference upstream builds succeeded"?